### PR TITLE
add four backend method tests,including 'sin','cos','copy' and 'power'

### PR DESCRIPTION
**Pull request by ritualOne**
the change allows us to test four new methods

### CODE REVIEW

File: tests/test_backends.py [hunk 1]
```diff
@@ -176,6 +176,16 @@ def test_backend_methods(backend):
...
+    np.testing.assert_allclose(
+        tc.backend.copy(arr),
+        arr,
+        atol=1e-5,
+    )
+    np.testing.assert_allclose(
+        tc.backend.power(arr,6),
+        np.power(arr,6),
+        atol=1e-5
+    )
```
Comment (refraction-ray) [inline tests/test_backends.py:188]: could these two methods tests also move to ``test_backend_method_2``? namely, whether currently pytorch backend supports these two methods

**Review by refraction-ray — CHANGES_REQUESTED** [review]
failed black now, please run ``black`` locally to ensure the code format

File: tests/test_backends.py (hunk 1 above)
Comment (ritualOne) [reply tests/test_backends.py]: yes,change will be committed soon

**Comment by refraction-ray** [comment]
closed as replaced by https://github.com/tencent-quantum-lab/tensorcircuit/pull/109

**Pull request closed without merging**